scripts/startup.sh.tpl

```diff
@@ -45,6 +51,16 @@ chmod 0600 /etc/vault.d/config.hcl
 # $$ is correct here because we are in terraform template
 sed -i "s/LOCAL_IP/$${LOCAL_IP}/g" /etc/vault.d/config.hcl
 
+# Pull IP info from gcloud if an internal load balancer is
+# required to avoid terraform cycle errors.
+LB_IP=${lb_ip}
+if [ '${create_external_load_balancer}' == 'false' ]; then
```

Comment:
Can you tell me more about the cycle you found? I don't like doing this on startup...

Reply:
Yeah I'm not a fan of this either, but it's because the instance template needs the internal load balancer IP to configure Vault, and we don't have the internal load balancer IP until it's created. But the internal load balancer can't be created without a managed group, which is dependent on the instance template. This works for external because we're creating the IP first. Definitely open to suggestions here.

Reply:
Can we hard-code it as a variable? `spec: loadBalancerIP` should accept an IP and so should Vault. We can use Terraform's cidr/ip helpers if needed, but picking like 10.x.y.z and allowing people to override seems better?

Reply:
Yeah that's a good call. Looking back on it, I'm not really sure why I didn't think of that. Makes it more consistent with the external scheme as well.